x.gemspec

```diff
@@ -10,25 +10,26 @@ Gem::Specification.new do |spec|
   spec.homepage = "https://sferik.github.io/x-ruby"
   spec.license = "MIT"
   spec.required_ruby_version = ">= 3.0"
+  spec.platform = Gem::Platform::RUBY
```

Comment:
I believe `"ruby"` is the default value. What’s the benefit of explicitly specifying it here?

Reply:
to indicate explicitely the gem will work on any platform and not only on a specific one like x86_64 that could be the case for a native gem